README.md

```diff
@@ -21,15 +21,27 @@ With this plugin you can share on instagram stories and facebook stories and als
 
 ##### For example:
 
-```
+```xml
         <manifest xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:tools="http://schemas.android.com/tools"
                 package="your package...">
 ```
+#### for android 11 and above Add this queries to AnroidManfist
+```xml
+<queries>
+		<!-- social share -->
+		<package android:name="com.instagram.android" />
+		<package android:name="com.twitter.android" />
+		<package android:name="com.whatsapp" />
+        	<package android:name="com.whatsapp.w4b" />
+		<package android:name="org.telegram.messenger" />
+<queries>
```

Comment:
</queries>

it should be a closing tag

Reply:
</queries>

Reply:
`</queries>`

Reply:
fixed